src/components/about/AboutIpld.js

```diff
@@ -3,21 +3,21 @@ import { Trans, withTranslation } from 'react-i18next'
 import Box from '../box/Box'
 import ipldLogoSrc from './ipld.svg'
 
-export const AboutIpld = () => {
+export const AboutIpld = ({ t }) => {
   return (
     <Box className='tl dib pa4 avenir measure-wide-l lh-copy dark-gray ba-l b--black-10'>
       <div className='tc'>
         <a className='link' href='https://ipld.io'>
           <img src={ipldLogoSrc} alt='IPLD' style={{ height: 60 }} />
         </a>
       </div>
-      <Trans i18nKey='AboutIpld.paragraph1'>
+      <Trans i18nKey='AboutIpld.paragraph1' t={t}>
```

Comment:
This was the fix for translations that did not load correctly.
I will never admit how long it took me to figure this out. :no_mouth: 

![problem-solver](https://user-images.githubusercontent.com/157609/83699437-4b33fe00-a604-11ea-9ef3-6cfc9e69b495.gif)

(When used as component, need to explicitly use `t` passed to component, rather than using the magically resolved one)

Reply:
I’d never have found that.